packages/turf-simplify/package.json

```diff
@@ -22,16 +22,23 @@
     "simplify"
   ],
   "author": "Turf Authors",
+  "contributors": [
+    "Stefano Borghi <@stebogit>"
+  ],
```

Comment:
Try including past contributors to this list when updating modules (@tmcw was heavily involved in the initial development of this module).

Click the **history** button on the `index.js` file.
https://github.com/Turfjs/turf/commits/b360f9beeec921c38468062fce893158e4a84988/packages/turf-simplify/index.js